Source/Manager/TaskManager/TaskManager.swift

```diff
@@ -77,6 +77,23 @@ class TaskManager {
                         DispatchQueue.main.async {
                             completion?(result)
                         }
+
+                        #if DEBUG
+                        if SDKConfiguration.shared.invalidateteAuthTokenAfterSuccessfullRequest {
```

Comment:
Should this be inside its own function to separate it more from the main code?

Reply:
🗡 😛